Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml

```diff
@@ -96,3 +99,51 @@
   - type: ContainerContainer
     containers:
       storagebase: !type:Container
+
+- type: entity
+  abstract: true
+  id: BaseBureaucraticStorageFill
+  components:
+  - type: StorageFill
+    contents:
+      - id: Pen
+        prob: 0.5
+      - id: PaperOffice
+      - id: PaperOffice
+        prob: 0.5
+      - id: PaperOffice
+        prob: 0.4
+        orGroup: Folder
```

Comment:
i assumed they used it like this because they wanted to guarantee one piece of paper, with a chance of spawning another one or not and a chance of spawning a third one or a folder which seems like a valid use since they wanted the filling to be very varied

Reply:
You can specify a minimum amount.

Reply:
Oh right, i forgot you can specify a range with maxAmmount, oops

Reply:
Like that?

`- id: PaperOffice 
amount: 1  
maxAmount: 2`

I'll fix that part day after tomorrow